cpp/src/gandiva/bitmap_accumulator_test.cc

```diff
@@ -38,7 +38,7 @@ void TestBitMapAccumulator::FillBitMap(uint8_t* bmap, int nrecords) {
 
   for (int i = 0; i < nbytes; ++i) {
     rand_r(&cur);
```

Comment:
Use `random_bytes` (https://github.com/apache/arrow/blob/master/cpp/src/arrow/test-util.h#L186) since it provides a deterministic seeded reproducible test.